### PR TITLE
Christian

### PR DESCRIPTION
**Pull request by Smell0ne**
okay n prof

### CODE REVIEW

File: Myweb/index.html [hunk 1]
```diff
@@ -102,8 +102,13 @@ <h1><span id="span1">The Canteen's Special Menu</span></h1>
         <tr align="center">
           <td>
             <button
+<<<<<<< HEAD
```
Comment (jonelpogi07) [inline Myweb/index.html]: clean

File: Myweb/index.html [hunk 2]
```diff
@@ -102,8 +102,13 @@ <h1><span id="span1">The Canteen's Special Menu</span></h1>
...
+              onclick="updatePage
+              ('orderedReceive.html', 'ChickenAdobo', 'Images1/chickenAdobo.png')
+=======
               onclick="updateDropdown
               ('orderedReceive.html', 'ChickenAdobo')
+>>>>>>> a55660f18be23986db43298d96247932d9edefad
```
Comment (jonelpogi07) [inline Myweb/index.html]: clean

File: Myweb/index.html [hunk 3]
```diff
@@ -113,8 +118,13 @@ <h1><span id="span1">The Canteen's Special Menu</span></h1>
 
           <td>
             <button
+<<<<<<< HEAD
```
Comment (jonelpogi07) [inline Myweb/index.html]: clean

File: Myweb/index.html [hunk 4]
```diff
@@ -306,9 +366,17 @@ <h1><span id="span1">The Canteen's Special Menu</span></h1>
...
+        function updatePage(targetPage, selectedOption, backgroundImage) {
+          const queryParams = new URLSearchParams();
+          queryParams.set("selectedOption", encodeURIComponent(selectedOption));
+          queryParams.set("backgroundImage", backgroundImage);
+          window.location.href = targetPage + "?" + queryParams.toString();
+=======
         function updateDropdown(targetPage, selectedOption) {
           window.location.href =
             targetPage + "?selectedOption=" + selectedOption;
+>>>>>>> a55660f18be23986db43298d96247932d9edefad
```
Comment (jonelpogi07) [inline Myweb/index.html]: clean

File: Myweb/index.html [hunk 5]
```diff
@@ -284,8 +334,13 @@ <h1><span id="span1">The Canteen's Special Menu</span></h1>
           </td>
           <td>
             <button
+<<<<<<< HEAD
+              onclick="updatePage
+              ('orderedReceive.html', 'KareKare', 'Images1/karekare.jpg')
+=======
               onclick="updateDropdown
               ('orderedReceive.html', 'Kare-Kare')
+>>>>>>> a55660f18be23986db43298d96247932d9edefad
```
Comment (jonelpogi07) [inline Myweb/index.html]: clean

File: Myweb/index.html (hunk 1 above)
Comment (Smell0ne) [reply Myweb/index.html]: Fenesh!

File: Myweb/index.html (hunk 2 above)
Comment (Smell0ne) [reply Myweb/index.html]: Fenesh!

File: Myweb/index.html (hunk 3 above)
Comment (Smell0ne) [reply Myweb/index.html]: Fenesh!

File: Myweb/index.html (hunk 5 above)
Comment (Smell0ne) [reply Myweb/index.html]: Fenesh!

File: Myweb/index.html (hunk 4 above)
Comment (Smell0ne) [reply Myweb/index.html]: Fenesh!

**Pull request merged**